src/containers/Dashboard/Dashboard.tsx

```diff
@@ -0,0 +1,66 @@
+/* eslint-disable global-require */
+// NOTE: disable global-require for dummy data, will be removed
+// once data is fed top-down
+
+import React from "react";
+import FilterBar from "../FilterBar/FilterBar";
+import { MovieProps } from "../../components/Movie/Movie";
+import MovieList from "../MovieList/MovieList";
+
+import styles from "./Dashboard.module.scss";
+
+// DATA
+
+const genres = ["All", "Documentary", "Comedy", "Horror", "Crime"];
```

Comment:
Consider creating a "constants" folder under "src/core" where all the main reusable non-react things will be stored.
"constants" will hold any number of file to your liking - as long as they contain all the hard-coded numbers/strings that you use throughout the app. The idea is to have a central place for all the constants to be able to easily reuse/change them.
Once all the constants are in "src/core/constants" don't forget to go through the app and use them where applicable.

Reply:
It's fine to just import them since it's more of a reuse matter than a design&structure.
Props are usually left for parent-child focused data and/or Dependency Injection things.